docs/content/release_notes/supported_storage.md

```diff
@@ -1,17 +1,16 @@
 # Supported storage systems
 
-IBM® block storage CSI driver 1.10.0 supports different IBM storage systems as listed in the following table.
+IBM® block storage CSI driver 1.11.0 supports different IBM storage systems as listed in the following table.
 
 |Storage system|Microcode version|
 |--------------|-----------------|
-|FlashSystem™ A9000|12.3.2.c or later|
-|FlashSystem A9000R|12.3.2.c or later|
 |Spectrum Virtualize™ family including Spectrum Virtualize as software only, Spectrum Virtualize for Public Cloud, SAN Volume Controller (SVC), and FlashSystem® family members built with Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx)|7.8.x, 8.2.x, 8.3.x, 8.4.x, 8.5.x|
 |DS8000® family|8.x and higher with same API interface|
 
 **Note:**
 
 - For the latest microcode storage support information, see the [Lifecycle and support matrix](https://www.ibm.com/docs/en/stg-block-csi-driver?topic=SSRQ8T/landing/csi_lifecycle_support_matrix.html).
+- Starting from CSI driver 1.11.0 release, FlashSystem A9000 and A9000R storage systems are no longer supported.
```

Comment:
I talked to @erantzabari and we don't want to explicitly write that we don't support A9000, can you remove this line?

Reply:
done